src/pages/VeSDL/index.tsx

```diff
@@ -468,7 +470,7 @@ export default function VeSDL(): JSX.Element {
                 )}
               </Typography>
             </Typography>
-            {!penaltyAmount.isZero() && leftTimeForUnlock && (
+            {!penaltyAmount.isZero() && leftTimeForUnlock && !isExpired && (
```

Comment:
These all mean the same thing, correct? If `isExpired`, then `leftTimeForUnlock <= 0`, and `penaltyAmount <= 0`. This can be simplified.

Reply:
Also `penaltyAmount` may be less than zero, so `penaltyAmount.isZero()` would be false. Better to use `penaltyAmount.lte(Zero)`

Reply:
Yes, but because of type script, if we don't check the leftTimeForUnlock, `secondsToHours(leftTimeForUnlock)` makes error even though we are check it on `!isExpired`.
That is the reason why I repeated same logic again.

And I agree to use `penaltyAmout.lte(Zero)`